client/app/common/services/teamStorageService.js

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) Trainline Limited, 2016-2017. All rights reserved. See LICENSE.txt in the project root for license information. */
+
+'use strict';
+
+(function () {
+  angular
+    .module('EnvironmentManager.common')
+    .factory('teamstorageservice', teamstorageservice);
+
+  teamstorageservice.$inject = ['localstorageservice'];
+
+  function teamstorageservice(localstorageservice) {
+    var key = 'em-selections-team';
+
+    return {
+      get: get,
+      set: set
+    };
+
+    function get(defaultValue) {
+      var value = '';
```

Comment:
Could this not be implemented as 

```javascript
return localstorageservice.exists(key) ? localstorageservice.get(key) : defaultValue
```

Reply:
Yours does look nicer. I went with this approach to make sure that even if they didn't provide a default value, we still fed something back if it didn't exist. I didn't want to write a ternary expression that included that logic too. Otherwise, yes I prefer the style of your comment above.